product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx

```diff
@@ -64,21 +66,21 @@ SELECT bdr.alter_table_conflict_detection('crdt_example',
 			'column_modify_timestamp', 'cts');
 ```
 
-And then insert a row with a value for this example.
+Insert a row with a value for this example:
 
 ```
 INSERT INTO crdt_example (id) VALUES (1);
 ```
 
-If you now issue, as before, the same SQL on two nodes at same time. 
+Suppose you now issue the same SQL on two nodes at same time as you did in the non-CRDT example: 
 
 ```sql
 UPDATE crdt_example
    SET counter = counter + 1    -- "reflexive" update
  WHERE id = 1;
 ```
 
-Once the changes are applied, you find that the counter has managed to concurrenct updates.
+Once the changes are applied, you find that the counter has managed two concurrent updates:
```

Comment:
Please verify this edit. The correction to concurrent is correct but I wanted to verify that to was meant to be two. Otherwise, a verb is missing.

Reply:
This does look right. Nice catch!